lib/fluent/plugin/out_http.rb

```diff
@@ -121,6 +127,36 @@ def configure(conf)
         end
         define_singleton_method(:format, method(:format_json_array))
       end
+
+      if @auth and @auth.method == :aws_sigv4
+        begin
+          require 'aws-sigv4'
+          require 'aws-sdk-core'
+        rescue LoadError
+          raise Fluent::ConfigError, "The aws-sigv4 and aws-sdk-core gems are required for aws_sigv4 auth. Run: 'gem install aws-sigv4 -v 1.8.0' and 'gem install aws-sdk-core -v 3.191'."
+        end
```

Comment:
I have some concern for installing XML parser. Because AWS requests are basically handled via XML responses. So, we need to indicate that XML parsers are also required to work for AWS SigV4 requests.
Should we add another require checks for XML parser gems such as nokogiri or rexml?

Reply:
@cosmo0920,

I considered this, but the AWS SDK already does this check and includes information on the available parsers. I think we should rely on theirs because otherwise we need to check for the same list that they have.

https://github.com/aws/aws-sdk-ruby/blob/a703da0525ac26d2ac4a68253b3718b6d86461b8/gems/aws-sdk-core/lib/aws-sdk-core/xml/parser.rb#L74-L76

Reply:
Ah, sounds reasonable.